pmd/defs.bzl

```diff
@@ -58,6 +58,12 @@ def _impl(ctx):
     arguments.add("--no-cache")
     arguments.add("--threads", ctx.attr.threads_count)
 
+    # Execution-result config
+    file_extension = "bat" if ctx.attr.is_windows else "sh"
```

Comment:
If you'd like to support windows I would add a windows github action so that we can continue to make sure that it works.